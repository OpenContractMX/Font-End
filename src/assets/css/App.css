* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

html {
  font-size: 62.5%;
}

body {
  background-color: #f2f4f4;
}

/*# sourceMappingURL=App.css.map */
