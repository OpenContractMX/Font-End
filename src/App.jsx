import React, { Fragment } from "react";
import { Header } from "./components/header";
import "../src/assets/css/App.css";

export const App = () => {
  return (
    <Fragment>
      <Header />
    </Fragment>
  );
};
