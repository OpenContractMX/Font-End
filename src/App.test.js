import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { App } from "./App";

it("render content", () => {
  const filter = { category: "Categoria", year: "AÃ±o" };
  const { asFragment } = render(<App filter={filter} />);
  expect(asFragment()).toMatchSnapshot();
});
