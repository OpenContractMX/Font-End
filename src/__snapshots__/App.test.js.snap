// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render content 1`] = `
<DocumentFragment>
  <header
    class="header"
    id="header"
  >
    <div
      class="header__container-logo"
    >
      <figure
        class="header__container-logo--logo"
      >
        <a
          href="/"
        >
          <img
            alt="logo"
            src="logo.png"
          />
        </a>
      </figure>
      <span
        class="header__container-logo--name"
      >
        OpenContractMX
      </span>
    </div>
    <nav
      class="navBar"
    >
      <i
        class="navBar__icon-menu"
      >
        <svg
          fill="none"
          height="1em"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <line
            x1="3"
            x2="21"
            y1="12"
            y2="12"
          />
          <line
            x1="3"
            x2="21"
            y1="6"
            y2="6"
          />
          <line
            x1="3"
            x2="21"
            y1="18"
            y2="18"
          />
        </svg>
      </i>
      <ul
        class="navBar__list "
      >
        <li
          class="nav__list--home"
        >
          <a
            class="list__home--name"
            href="/"
          >
            <i
              class="list__home--icon-home fas fa-home"
            />
            Inicio
          </a>
        </li>
        <li
          class="nav__list--statistic"
        >
          <a
            class="list__statistic--name"
            href="/statistic"
          >
            <i
              class="list__statistic--icon-chart fas fa-chart-line"
            />
            Estad√≠stica
          </a>
        </li>
        <li
          class="nav__list--about-us"
        >
          <a
            class="list__about-us--name"
            href="/about-us"
          >
            <i
              class="list__about-us--icon-team fas fa-people-carry"
            />
            Sobre Nosotros
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <main>
    <section
      class="home"
    >
      <h1
        class="home__title"
      >
        OpenContractMX
      </h1>
      <h2
        class="home__subtitle"
      >
        Bienvenido, en esta plataforma podr√°s encontrar informaci√≥n y estad√≠sticas de inter√©s referente a Contratos Federales de M√©xico.
      </h2>
      <p
        class="home__paragraph--use"
      >
        C√≥mo puedes usar nuestra plataforma?
      </p>
      <p
        class="home__paragraph"
      >
        1. Puedes filtrar por Categor√≠a y a√±o.
      </p>
      <div
        class="home__container-example"
      >
        <figure
          class="home__container-example--img"
        >
          <img
            alt="Imagen ilustrativa paso 1"
            src="filtrar.png"
          />
        </figure>
      </div>
      <p
        class="home__paragraph"
      >
        2. Observar un total de contratos e inversi√≥n.
      </p>
      <div
        class="home__container-example"
      >
        <figure
          class="home__container-example--img"
        >
          <img
            alt="Imagen ilustrativa paso 2"
            src="cardDatosContratos.png"
          />
        </figure>
      </div>
      <p
        class="home__paragraph"
      >
        3. Ver todos los contratos de manera detallada por mes o por trimestre.
      </p>
      <div
        class="home__container-example"
      >
        <figure
          class="home__container-example--img"
        >
          <img
            alt="Imagen ilustrativa paso 3"
            src="verTodos.png"
          />
        </figure>
      </div>
      <p
        class="home__paragraph"
      >
        4. Puedes ver el Top 10 de contratos por gastos.
      </p>
      <div
        class="home__container-example"
      >
        <figure
          class="home__container-example--img"
        >
          <img
            alt="Imagen ilustrativa paso 4"
            src="top10.png"
          />
        </figure>
      </div>
      <p
        class="home__paragraph"
      >
        5. Observar el n√∫mero de contratos por mes.
      </p>
      <div
        class="home__container-example"
      >
        <figure
          class="home__container-example--img"
        >
          <img
            alt="Imagen ilustrativa paso 5"
            src="contXmes.png"
          />
        </figure>
      </div>
      <p
        class="home__paragraph"
      >
        6. Y un tiempo de ejecuci√≥n promedio para estos contratos de la categor√≠a detallada.
      </p>
      <div
        class="home__container-example"
      >
        <figure
          class="home__container-example--img"
        >
          <img
            alt="Imagen ilustrativa paso 7"
            src="ejecProm.png"
          />
        </figure>
      </div>
      <p
        class="home__paragraph"
      >
        Toda la informaci√≥n que puede ser obtenida por esta aplicaci√≥n fue rescatada y procesada con los datos obtenidos el 14 de mayo del 2021, en el portal de la 
        <a
          class="home__paragraph--link"
          href="https://datos.gob.mx/busca/dataset/concentrado-de-contrataciones-abiertas-de-la-apf-shcp/resource/aa36c461-a1d3-4373-bfb5-ca479ed86f26?inner_span=True"
          rel="noreferrer"
          target="_blank"
        >
           SHCP
        </a>
        (Secretaria de Hacienda y Cr√©dito P√∫blico).
      </p>
      <div
        class="home__container-chart"
      >
        <figure
          class="home__container-chart--logo"
        >
          <img
            alt="Imagen del Home"
            src="chartHome.png"
          />
        </figure>
      </div>
    </section>
  </main>
  <footer
    class="footer"
  >
    <p
      class="footer--Copyright"
    >
      Copyright¬© Made with üß° from LATAM to the üåé
    </p>
  </footer>
</DocumentFragment>
`;
